modules/social_features/social_profile/social_profile.install

```diff
@@ -460,3 +460,17 @@ function social_profile_update_11401(): string {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Fix missed filter block on the all-members page.
+ */
+function social_profile_update_11402(): string {
```

Comment:
@volodymyr-sydor Please change update hook number to start with `11601` - the same as milestone.

Reply:
done

Reply:
@volodymyr-sydor, sorry, since we don't support 11.6 anymore - could you increase it to `11701`?

Reply:
done